pkg/webui/console/views/device-add/manual/form/advanced-settings.js

```diff
@@ -0,0 +1,299 @@
+  }
+})
+
+const timeoutEncode = value => (Boolean(value) ? `${value}s` : value)
+const timeoutDecode = value => (Boolean(value) ? RegExp(/\d+/).exec(value)[0] : value)
+
+const allClassOptions = [
+  { label: m.classA, value: DEVICE_CLASS_MAP.CLASS_A },
+  { label: m.classB, value: DEVICE_CLASS_MAP.CLASS_B },
+  { label: m.classC, value: DEVICE_CLASS_MAP.CLASS_C },
+  { label: m.classBandC, value: DEVICE_CLASS_MAP.CLASS_B_C },
+]
+const multicastClassOptions = allClassOptions.filter(
+  ({ value }) => value !== DEVICE_CLASS_MAP.CLASS_A,
+)
+
+const AdvancedSettingsSection = props => {
+  const {
+    nsEnabled,
+    jsEnabled,
+    onActivationModeChange,
+    onDeviceClassChange,
+    deviceClass,
+    activationMode,
+    onDefaultNsSettingsChange,
+    defaultNsSettings,
+  } = props
+
+  const isOTAA = activationMode === ACTIVATION_MODES.OTAA
+  const isABP = activationMode === ACTIVATION_MODES.ABP
+  const isMulticast = activationMode === ACTIVATION_MODES.MULTICAST
+  const isClassB =
+    deviceClass === DEVICE_CLASS_MAP.CLASS_B_C || deviceClass === DEVICE_CLASS_MAP.CLASS_B
+  const isClassC =
+    deviceClass === DEVICE_CLASS_MAP.CLASS_B_C || deviceClass === DEVICE_CLASS_MAP.CLASS_C
+
+  const [externalServers, setExternalServer] = React.useState(false)
+  const handleExternalServers = React.useCallback(
+    () => setExternalServer(external => !external),
+    [],
+  )
+
+  const handleDefaultNsSettings = React.useCallback(
+    evt => onDefaultNsSettingsChange(evt.target.checked),
+    [onDefaultNsSettingsChange],
+  )
+
+  return (
+    <Form.CollapseSection
+      className={style.advancesSection}
+      id="advanced-settings"
+      title={m.advancedSectionTitle}
+    >
+      <Form.Field
+        title={sharedMessages.activationMode}
+        name="_activation_mode"
+        component={Radio.Group}
+        disabled={!nsEnabled && !jsEnabled}
+        required={nsEnabled || jsEnabled}
+        tooltipId={tooltipIds.ACTIVATION_MODE}
+        onChange={onActivationModeChange}
+      >
+        <Radio label={sharedMessages.otaa} value={ACTIVATION_MODES.OTAA} disabled={!jsEnabled} />
+        <Radio label={sharedMessages.abp} value={ACTIVATION_MODES.ABP} disabled={!nsEnabled} />
+        <Radio
+          label={sharedMessages.multicast}
+          value={ACTIVATION_MODES.MULTICAST}
+          disabled={!nsEnabled}
+        />
+      </Form.Field>
+      <Form.Field
+        title={isMulticast ? m.multicastClassCapabilities : messages.classCapabilities}
+        required={isMulticast}
+        name="_device_class"
+        component={Select}
+        onChange={onDeviceClassChange}
+        options={isMulticast ? multicastClassOptions : allClassOptions}
+        tooltipId={tooltipIds.CLASSES}
+      />
+      <Form.Field
+        label={messages.defaultNetworksSettings}
+        name="_default_ns_settings"
+        component={Checkbox}
+        onChange={handleDefaultNsSettings}
+        tooltipId={tooltipIds.NETWORK_RX_DEFAULTS}
```

Comment:
Note that this tooltip is not shown because the field does not use a title. So to remedy quickly, let's use the title `Network defaults` for this. This should be replaced then as soon as we can add tooltips to checkboxes as well.